core/src/internal/image_utils.cpp

```
@@ -96,53 +97,54 @@ void gaussian_blur_fft(uint8_t *image, size_t width, size_t height, double sigma
         p.blue = 255 - p.blue;
         p.green = 255 - p.green;
     }
 
-    const auto &modified = img.getData();
+    const auto& modified = img.getData();
     std::memcpy(ptr, modified.data(), modified.size() * sizeof(ImageLib::RGBAPixel<uint8_t>));
 }
 
-void threshold_image(uint8_t *ptr, const int width, const int height, const int num_thresholds) {
-    const uint8_t REGION_SIZE(255 / num_thresholds);  // Size of buckets per colour
+void threshold_image(uint8_t* ptr, const int width, const int height, const int num_thresholds) {
```

Cppcheck (2.20.0)

[style] The function 'threshold_image' is never used.

(unusedFunction)